mission_control/navigator_launch/launch/cameras.launch

```diff
@@ -0,0 +1,80 @@
+<launch>
+  <!-- This is the default pointgrey launch file with our id's and our names  -->
+
+  <arg name="front_camera_serial" default="15223362" />
+  <arg name="front_camera_calibrated" default="0" />
+  <arg name="down_camera_serial" default="10282860" />
+  <arg name="down_camera_calibrated" default="0" />
```

Comment:
Also, the Point Grey drivers absolutely suck, I spent months trying to run more than one camera at the same time and it never worked. The new drivers they put out might work better but I don't care to find out. The sub runs 1394, and last I checked the boat was also running 1394.